src/main/resources/application.properties

```diff
@@ -1,3 +1,13 @@
+quarkus.openshift.route.expose=true
+quarkus.openshift.labels."app"=wildfly-bot
+quarkus.openshift.env.vars.QUARKUS_GITHUB_APP_APP_NAME=wildfly-bot
```

Comment:
I think we can drop this property according to the docs. Also we might update README.md as we set the secrets name. User won't need to override it and less configuration for him to do.

Reply:
Env or sys prop still overrides the properties file

Reply:
Yes, but the `QUARKUS_GITHUB_APP_APP_NAME` should not be set. I expect `main` branch should be run only under `prod` profile. [Docs](https://docs.quarkiverse.io/quarkus-github-app/dev/create-github-app.html) source I forgot to post last comment.